src/api/libcellml/variable.h

```diff
@@ -39,6 +39,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     Variable& operator=(Variable n); /**< Assignment operator */
 
     /**
+     * @brief The INTERFACE_TYPES enum.
+     * Enum describing the interface types that are available for
+     * a given Variable.
+     */
+    enum INTERFACE_TYPES
+    {
+        INTERFACE_TYPE_UNDEFINED,
```

Comment:
Sure, but if it doesn't have an interface, then it means that it effectively has an interface of ```none```, no?

Personally, I find that ```INTERFACE_TYPE_UNDEFINED``` confusing. To me, we can't have an undefined interface, not least since none is specified then it effectively means that we have an interface of ```none```.